feature/src/main/java/com/teamdontbe/feature/posting/PostingFragment.kt

```diff
@@ -217,6 +188,33 @@ class PostingFragment : BindingFragment<FragmentPostingBinding>(R.layout.fragmen
         }
     }
 
+    private fun FragmentPostingBinding.updateProgress(
+        progressDrawableResId: Int,
+        textLength: Int,
+        backgroundTintResId: Int,
+        textColorResId: Int,
+        clickListener: () -> Unit,
+    ) {
+        pbPostingInput.progressDrawable =
+            context?.drawableOf(progressDrawableResId)
```

Comment:
```suggestion
            drawableOf(progressDrawableResId)
```
P2 : ContextExt 말고 FragmentExt 확장함수 쓰면 됨!
`import com.teamdontbe.core_ui.util.fragment.drawableOf`

Reply:
해놔도 있는지 모르는.. 적용했어용 ㅎㅎ